### PR TITLE
[ReadManhwa] add website support

### LINKED ISSUE #1503: Site Request: Readmanhwa
May I request this site to be added on HakuNeko?

https://readmanhwa.com/en/

Thank you~

### PR DESCRIPTION
**Pull request by manh21**
Resolve #1503

### CODE REVIEW

File: src/web/mjs/connectors/ReadManhwa.mjs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                id: el.slug+'&&%@'+manga.id,
+                title: el.name,
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const ch = chapter.id.split('&&%@');
+        const chapter_id = ch[0];
+        const manga_id = ch[1];
```
Comment (ronny1982) [inline src/web/mjs/connectors/ReadManhwa.mjs:59]: May use `chapter.manga.id`

File: src/web/mjs/connectors/ReadManhwa.mjs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    async _getChapters(manga) {
+        const uri = new URL(`/api/comics/${manga.id}/chapters`, this.url);
+        uri.searchParams.set('nsfw', true);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchJSON(request);
+        return data.map(el => {
+            return {
+                id: el.slug+'&&%@'+manga.id,
```
Comment (27136773) [inline src/web/mjs/connectors/ReadManhwa.mjs]: Adding `manga.id` seems redundant

File: src/web/mjs/connectors/ReadManhwa.mjs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                title: el.name,
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const ch = chapter.id.split('&&%@');
+        const chapter_id = ch[0];
+        const manga_id = ch[1];
+        const uri = new URL(`/api/comics/${manga_id}/${chapter_id}/images`, this.url);
```
Comment (27136773) [inline src/web/mjs/connectors/ReadManhwa.mjs]: May use `chapter.manga.id` and `chapter.id` instead

**Review by 27136773 — APPROVED** [review]
Looks good, minor improvements possible

**Pull request merged**